config.default

```diff
@@ -33,6 +33,16 @@ referenceRunName = None
 # MPAS-Analysis)
 preprocessedReferenceRunName = None
 
+[execute]
+## options related to executing parallel tasks
+
+# the number of parallel tasks (1 means tasks run in serial, the default)
+parallelTaskCount = 1
+
+# Prefix on the commnd line before a parallel task (e.g. 'srun -n 1 python')
+# Default is no prefix (run_analysis.py is executed directly)
+commandPrefix = 
```

Comment:
I have now added two job scripts, a default one in `configs` and an Edison-specific one in `configs/edison`.  I also updated the README files to give instructions on how to modify and use these parallel scripts.

Similar scripts should be added for other machines, but I would suggest we debug these first.